Packages/vcs/vcs/isoline.py

```diff
@@ -619,28 +619,60 @@ def _setlinecolors(self, value):
         self._linecolors = value
     linecolors = property(_getlinecolors, _setlinecolors)
 
-    def _getline(self):
-        return self._line
+    def _getlinetypes(self):
+        return self._linetypes
 
-    def _setline(self, value):
+    def _setlinetypes(self, value):
         if value is not None:
-            value, cvalue, wvalue = VCS_validation_functions.checkLinesList(
+            value = VCS_validation_functions.checkLineTypeList(
                 self,
-                'line',
+                'linetypes',
                 value)
-        self._line = value
-        try:
-            if self.linecolors is None or len(self.linecolors) < len(cvalue):
-                self.linecolors = cvalue
-        except:
-            pass
-        try:
-            if self.linewidths is None or len(self.linewidths) < len(wvalue):
-                self.linewidths = wvalue
-        except:
-            pass
-
-    line = property(_getline, _setline)
```

Comment:
@doutriaux1 @doutriaux1 Makes sense. The line interface will still have the bug @chaosphere2112 reported. We won't do any fixes there. Maybe remove it in the next version.